prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.html

```diff
@@ -49,7 +49,6 @@
   </app-page-subheader>
 
   <app-enrollee-review *ngIf="enrolment"
-                       [showEditRedirect]="currentStatus === EnrolmentStatus.EDITABLE"
```

Comment:
This is hiding edit for the whole overview it looks like.

Reply:
> This is hiding edit for the whole overview it looks like.

Added this line back.